### PR TITLE
Slight edit to active_support/rails docs

### PR DESCRIPTION
**Pull request by maclover7**
[ci skip]

### CODE REVIEW

File: activesupport/lib/active_support/rails.rb [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-#
 # Rails components cherry pick from Active Support as needed, but there are a
-# few features that are used for sure in some way or another and it is not worth
-# putting individual requires absolutely everywhere. Think blank? for example.
+# number of features that are regularly needed (ex: blank?), and this serves as
+# a way to reduce the total number of requires.
 #
 # This file is loaded by every Rails component except Active Support itself,
-# but it does not belong to the Rails public interface. It is internal to
+# but it is not apart of Rails' public API. It is internal to
```
Comment (kaspth) [inline activesupport/lib/active_support/rails.rb]: Doesn't `apart` signify away from? How about just `part`?

File: activesupport/lib/active_support/rails.rb [hunk 2]
```diff
@@ -1,17 +1,15 @@
-# This is private interface.
```
Comment (kaspth) [inline activesupport/lib/active_support/rails.rb]: Not sure if this is correct :)

If I understand this file correctly, it's to setup common Rails requires across components. Not something for apps to piggy back on; they've got the top level `'active_support'` require 😁

Comment (kaspth) [inline activesupport/lib/active_support/rails.rb]: Ah it's still expressed below. Curious why we'd want a warning up top and then not keep the whole privacy bit there as well.

File: activesupport/lib/active_support/rails.rb (hunk 1 above)
Comment (kaspth) [inline activesupport/lib/active_support/rails.rb]: Would also merge this and the next sentence.

**Comment by maclover7** [comment]
👍 updated per comments

**Comment by kaspth** [comment]
Reading through this some more these proposed changes don't improve the existing docs much — and this is a private file, so I'd rather keep the git history here.

Feel free to reopen if you disagree, @maclover7 😁

**Comment by maclover7** [comment]
No problem, I understand that point of view :)

**Pull request closed without merging**